meta-balena-common/recipes-connectivity/openvpn/files/openvpn.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=OpenVPN
-Requires=prepare-openvpn.service bind-etc-openvpn.service
-After=syslog.target network.target prepare-openvpn.service bind-etc-openvpn.service time-sync.target
+Requires=prepare-openvpn.service bind-etc-openvpn.service time-sync-https-wait.target
+After=syslog.target network.target prepare-openvpn.service bind-etc-openvpn.service time-sync-https-wait.target
```

Comment:
As mentioned on FD, what is the state of `openvpn.service` if `apiEndpoint` is not set in `config.json`?
OS suite: https://github.com/balena-os/meta-balena/blame/master/tests/suites/os/suite.js#L66
HUP suite (failing): https://github.com/balena-os/meta-balena/blob/master/tests/suites/hup/suite.js#L306

Should be fairly easy to test with an unmanaged image.
1. Flash unmanaged development image from https://www.balena.io/os/
2. `hostapp-update` to a hostapp from this PR
3. Confirm that rollback services run on reboot and clean up breadcrumbs

Reply:
I've tested with an unmanaged image and the easiest solution is to just skip the HTTPS header synchronisation if the connectivity URL is empty.